pkg/identity/identity.go

```diff
@@ -112,25 +114,31 @@ func (h *identityHandler) GetX509Cert() (*zts.InstanceIdentity, []byte, error) {
 		return nil, nil, err
 	}
 
+	provider := zts.ServiceName(h.config.ProviderService)
+	domain := zts.DomainName(h.domain)
+	service := zts.SimpleName(h.service)
+	attestationData := string(saToken)
+	csrPEMString := string(csrPEM)
+
 	if h.config.Init {
+		log.Debugf("Sending PostInstanceRegisterInformation request to ZTS, Provider: %s, Domain: %s, Service: %s, Attestation Data: %s, CSR: %s", provider, domain, service, attestationData, csrPEM)
```

Comment:
can we not log the attestationData since it could be used for replay attacks?

Reply:
@prabushyam Hi Prabu, since it's set as a debug level, I suppose it wouldn't be a risk to print. I would also think it's useful for some troubleshooting cases.

Reply:
@prabushyam Are you okay with attestationData being printed in debug mode?